docs/resources/azure_iothub.md

```diff
@@ -0,0 +1,91 @@
+---
+title: About the azure_iothub Resource
+platform: azure
+---
+
+# azure_iothub
+
+Use the `azure_iothub` InSpec audit resource to test properties of an Azure IoT hub within a Resource Group.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used .
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+The `resource_group` and `name` must be given as a parameter.
```

Comment:
This isn't strictly true based on the Parameters explanation. They could also just provide the `resource_id` right?

Reply:
Would this explain this?
```
The `resource_group` and `name` or the `resource_id` must be given as a parameter.
```

Reply:
Fixed.